openadmet/models/registries.py

```diff
@@ -13,10 +13,12 @@
 # featurizers
 from openadmet.models.features.molfeat_fingerprint import *  # noqa: F401 F403
 from openadmet.models.features.molfeat_properties import *  # noqa: F401 F403
-from openadmet.models.architecture.chemprop import *  # noqa: F401 F403
+from openadmet.models.features.mtenn import *  # noqa: F401 F403
```

Comment:
Just a note here for future reference, you have to add featurizers and models to be imported here for it to register properly for Anvil